workspace-manager-clienttests/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
     // Apply the application plugin to add support for building a CLI application in Java.
     id 'application'
     // Terra Test Runner Plugin
-    id 'bio.terra.test-runner-plugin' version '0.0.3-20210311.184005-30'
+    id 'bio.terra.test-runner-plugin' version '0.0.3-20210316.131233-31'
```

Comment:
why not just take the latest?

Reply:
Updated version to `0.0.3-SNAPSHOT` which should pull latest version.